services/device-registry/src/test/java/org/eclipse/hono/deviceregistry/DeviceRegistryRestServerTest.java

```diff
@@ -0,0 +1,430 @@
+/**
+ * Copyright (c) 2017 Bosch Software Innovations GmbH.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Bosch Software Innovations GmbH - initial creation
+ */
+package org.eclipse.hono.deviceregistry;
+
+import static org.eclipse.hono.service.http.HttpEndpointUtils.CONTENT_TYPE_JSON;
+import static org.eclipse.hono.util.RegistrationConstants.REGISTRATION_ENDPOINT;
+import static org.eclipse.hono.util.RequestResponseApiConstants.FIELD_DEVICE_ID;
+
+import org.eclipse.hono.config.ServiceConfigProperties;
+import org.eclipse.hono.config.SignatureSupportingConfigProperties;
+import org.eclipse.hono.service.registration.RegistrationAssertionHelperImpl;
+import org.eclipse.hono.service.registration.RegistrationHttpEndpoint;
+import org.junit.*;
+import org.junit.runner.RunWith;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+
+/**
+ * Tests {@link DeviceRegistryRestServer}.
+ */
+@RunWith(VertxUnitRunner.class)
+public class DeviceRegistryRestServerTest {
+
+    private static final String HOST = "localhost";
+
+    private static final String TENANT = "testTenant";
+    private static final String DEVICE_ID = "testDeviceId";
+
+    private static Vertx vertx;
+    private static FileBasedRegistrationService registrationService;
+    private static DeviceRegistryRestServer deviceRegistryRestServer;
+
+    @BeforeClass
+    public static void setUp(final TestContext context) throws Exception {
+        vertx = Vertx.vertx();
```

Comment:
throws Exception : IMHO can be deleted?
Or is there a good reason for declaring them at all methods in this file?

Reply:
It wasn't added on purpose, the IDEA template for adding a test method included it. 
I'll remove them.